src/Select.js

```diff
@@ -86,7 +88,8 @@ var Select = React.createClass({
 			options: this.props.options,
 			isFocused: false,
 			isOpen: false,
-			isLoading: false
+			isLoading: false,
+			onReadOnlyMode: this.props.listReadOnlyMode
```

Comment:
this isn't an event right? I got confused by the "on" part
not a huge deal but I think it would be better to rename it